rules/ml/ml_linux_anomalous_sudo_activity.toml

```diff
@@ -0,0 +1,65 @@
+[metadata]
+creation_date = "2020/09/03"
+ecs_version = ["1.5.0"]
```

Comment:
1.6.0?

Reply:
they were tested on 1.5.0, so that is the minimum version

Reply:
if 1.6.0 doesn't implement any breaking changes, then these should be on the latest version.